src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DayRent = 40;
+  let totalRent = 0;
```

Comment:
Prefer using `const` over `let` for `totalRent` as it is not re-assigned but calculated conditionally.